config/environments/development.rb

```diff
@@ -34,4 +34,5 @@
   config.assets.debug = true
 
   BetterErrors::Middleware.allow_ip! '192.168.0.1/16'
+  Rails.application.routes.default_url_options[:host] = 'hydranorthdev.library.ualberta.ca'
```

Comment:
These files are managed by ansible.  We need to make sure this change gets back into the Ansible templates.